src/domain.py

```diff
@@ -111,12 +184,14 @@ def build_juju_pod_spec(app_name,
                                           advertised_port=advertised_port,
                                           monitor_k8s=monitor_k8s)
 
+    prometheus_cli_args = PrometheusServiceConfig(charm_config)
```

Comment:
Ah, yeah, the incongruence between the variable name and the class name suggests `PrometheusCLIArgs` is better or if a function, `build_prometheus_cli_args()`

Reply:
Indeed. I will fix that.